### DISCUSSION TITLE
Urgent Help Needed: Accidentally Disconnected GitHub Account from Vercel and Can't Access Projects

### DISCUSSION THREAD

**Discussion by irmakcosarsahna** [vercel/community#3374] [Feature Question] [answered]
### Question

Hello,

I accidentally disconnected my GitHub account from Vercel, and now I can't log in to my Vercel account. This has resulted in several of my projects being left inaccessible on Vercel. Additionally, I have a pro membership and host several corporate websites there. Could you please assist me in recovering my account?

Thank you.

**Comment by amyegan** [comment, marked as answer]
Hey @irmakcosarsahna. You should be able to [log in using your email address](https://vercel.com/login/email) and reconnect your GitHub account. Do you see any error message or other problems when trying that?

**Reply by irmakcosarsahna** [reply]
First of all, thank you. I logged in to GitHub with the wrong email address. There was confusion because I have 4 different email addresses associated with it. However, I resolved my issue by logging in with the correct email. Thank you again.
